code/game/gamemodes/clock_cult/clock_items.dm

```diff
@@ -599,7 +599,7 @@
 	icon_state = "clockwork_cuirass"
 	w_class = 4
 	body_parts_covered = CHEST|GROIN|LEGS
-	armor = list(melee = 80, bullet = 50, laser = -15, energy = 0, bomb = 35, bio = 0, rad = 0)
+	armor = list(melee = 80, bullet = 50, laser = -15, energy = 5, bomb = 35, bio = 0, rad = 0, fire = 50, acid = 50)
```

Comment:
I'm not sure what the values for these actually mean but all the clockwork armor should be fire and acid proof because it's magic clock armor.